### PR TITLE
Ecoreling db updates tv 230113

### PR DESCRIPTION
**Pull request by astridberndsen**
The norms for landscape and water (on peat soils) were updated (now 0.05 instead of 0). As a consequence the aims based on these new norms change. This means the output of er_main now also includes values for thresholds for landscape_bronze and landscape_silver. 

Also the threshold values on farm level were updated for bronze, silver and gold d into 15, 22.5 and 40 respectively.

The er_measures table was extended.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# BBWPC v.0.10.18
+
+## Changed
+* ecoregeling norms for landscape and water (on peat soil) were changed from 0 to 0.05 in `er_scoring`
+* in `er_farm_aim` the aims for landscape and water (on peat soils) got aims based on the new norms and were not set to 0 anymore
+* output of `er_main` now also includes values for `$farm$s_er_landscape_bronze` and `$farm$s_er_landscape_silver`
+* threshold values for bronze, silver and gold were changed into 15, 22.5 and 40 respectively in `er_farm_aim` 
+* the `er_measures` table was extended from 7720 rows to 10457 rows
```
Comment (gerardhros) [inline NEWS.md:8]: did you check that all the brp codes actually occur in pandex (the crop codes that we currently allow)

**Review by gerardhros — COMMENTED** [review]
seems a small update. nice done.
one question to ensure that the crop codes added are all allowed.

File: NEWS.md (hunk 1 above)
Comment (astridberndsen) [reply NEWS.md:8]: The brp_codes in the currently used er_measures table are exactly the same (also same number) as the brp codes in pandex. However, the source csv includes also brp_codes higher than 7000. Those three brp_codes are removed from er_measures.

**Review by gerardhros — APPROVED** [review]

File: NEWS.md (hunk 1 above)
Comment (gerardhros) [reply NEWS.md:8]: that makes sense. thanks

**Pull request merged**